packages/portfolio-contract/test/snapshots/portfolio.flows.test.ts.md

```diff
@@ -1438,3 +1438,172 @@ Generated by [AVA](https://avajs.dev).
+        },
+      },
+      {
+        _cap: 'seat',
+        _method: 'exit',
+        completion: undefined,
+      },
+    ]
+
+> Under "published", the "ymax0.portfolios" node is delegated to ymax.
+> The example below illustrates the schema of the data published there.
+> 
+> See also board marshalling conventions (_to appear_).
+
+    [
+      [
+        'published.ymax0.portfolios.portfolio1',
+        {
+          accountIdByChain: {
+            Avalanche: 'eip155:43114:0x126cf3AC9ea12794Ff50f56727C7C66E26D9C092',
+            agoric: 'cosmos:agoric-6:agoric11014',
+            noble: 'cosmos:noble-5:noble11028',
+          },
+          flowCount: 1,
+          positionKeys: [
+            'Beefy_re7_Avalanche',
+          ],
+        },
+      ],
+      [
+        'published.ymax0.portfolios.portfolio1.flows.flow1',
+        {
+          amount: {
+            brand: Object @Alleged: USDC brand {},
+            value: 300n,
+          },
+          dest: 'Beefy_re7_Avalanche',
+          how: 'Beefy',
+          src: 'eip155:43114:0x126cf3AC9ea12794Ff50f56727C7C66E26D9C092',
+          step: 4,
+        },
+      ],
+      [
+        'published.ymax0.portfolios.portfolio1.positions.Beefy_re7_Avalanche',
+        {
+          accountId: 'eip155:43114:0x126cf3AC9ea12794Ff50f56727C7C66E26D9C092',
+          netTransfers: {
+            brand: Object @Alleged: USDC brand {},
+            value: 300n,
+          },
+          protocol: 'Beefy',
+          totalIn: {
+            brand: Object @Alleged: USDC brand {},
+            value: 300n,
+          },
+          totalOut: {
+            brand: Object @Alleged: USDC brand {},
+            value: 0n,
+          },
+        },
+      ],
+    ]
+
+> decoded calls
+
+    [
+      {
+        args: [
+          '0xc3d688B66703497DAA19211EEdff47f25384cdc3',
+          300n,
+        ],
+        functionName: 'approve',
+      },
+      {
+        args: [
+          300n,
+        ],
+        functionName: 'deposit',
```

Comment:
`depositAll` is the method I saw in [tx/0xc2df6...](https://snowtrace.io/tx/0xc2df61572c7651c9ec7a220fde863543f253c52d0988ccd46c609690501366dd?chainid=43114).

An example `deposit` tx sure would be nice.